frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -51,40 +51,111 @@ function MethodologyTab() {
             </Grid>
             <Grid item xs={12} className={styles.MethodologyQuestionAndAnswer}>
               <h2 className={styles.MethodologyQuestion}>
-                What are the limitations of the data?
+                What are the limitations of the tracker?
               </h2>
               <div className={styles.MethodologyAnswer}>
+                <h3 className={styles.MethodologySubheaderText}>Data</h3>
+
+                <h4 className={styles.MethodologySubsubheaderText}>COVID-19</h4>
                 <p>
-                  Unfortunately, with these publically available data sets,
-                  there are crucial pieces missing, including but not limited
-                  to: comprehensive city-, census tract-, and county-level data;
-                  comprehensive race and ethnicity breakdowns; comprehensive
-                  gender and age breakdowns by county, etc.
+                  {/* TODO - this should really be a button... */}
+                  For a description of some of the gaps in COVID-19 data, please
+                  see the "What Data Are Missing" section on the "Explore the
```

Comment:
Add link?

Reply:
Planning to submit this first and then add it after (somewhat dependent on another PR)